src/coverlet.core/Helpers/InstrumentationHelper.cs

```diff
@@ -136,15 +135,15 @@ public bool EmbeddedPortablePdbHasLocalSource(string module, out string firstNot
         public bool PortablePdbHasLocalSource(string module, out string firstNotFoundDocument)
         {
             firstNotFoundDocument = "";
-            using (var moduleStream = File.OpenRead(module))
+            using (var moduleStream = _fileSystem.OpenRead(module))
             using (var peReader = new PEReader(moduleStream))
             {
                 foreach (var entry in peReader.ReadDebugDirectory())
                 {
                     if (entry.Type == DebugDirectoryEntryType.CodeView)
                     {
                         var codeViewData = peReader.ReadCodeViewDebugDirectoryData(entry);
-                        using FileStream pdbStream = new FileStream(codeViewData.Path, FileMode.Open);
+                        using FileStream pdbStream = _fileSystem.NewFileStream(codeViewData.Path, FileMode.Open) as FileStream;
```

Comment:
we can keep Stream remove as FileStream, we don't expect null from this api and other api below accept Stream

Reply:
Done